File: PYTHON/PART_1/File/Write.py
try:
  file=open('text.txt','w')
  file.write("Hi Hello")
  file=open('text.txt','r')
  line=file.read()
  print(line)  
except FileNotFoundError as e:
  print("File is not there")  
  print('error :',e)
else:
  file.close()
  print("success writed... ")
print('this is print line by line and Your charactors')
print('readline(Your charactor)')    

File: C++/Core/File/write.cpp
#include <iostream>
#include<fstream>
using namespace std;
int main(){
  cout<<"---------------------------------"<<endl;
  ofstream o("text.txt",ios::trunc);
  o<<"Hi hello"<<endl;
  o<<"i am Mohan";
  o.close();

  cout<<""<<endl;
  cout<<"---------------------------------"<<endl;
  return 0;
}